src/pepr/operator/controllers/istio/virtual-service.ts

```diff
@@ -105,10 +117,10 @@ export async function virtualService(pkg: UDSPackage, namespace: string) {
 }
 
 export function generateVSName(pkg: UDSPackage, expose: Expose) {
-  const { gateway = Gateway.Tenant, host, port, service, match, description } = expose;
+  const { gateway = Gateway.Tenant, host, port, service, description, advancedHTTP } = expose;
 
   // Ensure the resource name is valid
-  const matchHash = match?.flatMap(m => m.name).join("-") || "";
+  const matchHash = advancedHTTP?.match?.flatMap(m => m.name).join("-") || "";
```

Comment:
Small nit: this section doesn't handle the deprecated `expose.match` I don't think?